docs/onboarding/VPN.md

```diff
@@ -0,0 +1,32 @@
+## General documenation about connecting to Boulder office's VPN
+
+**Client configuration:**
+
+1. Create a public-private key pair: `umask 077 && wg genkey | tee privatekey | wg pubkey > publickey`
+2. Create a wireguard.conf file following the wireguard-client-config below
+3. Provide the public key to Barnabas#1342 on discord.  
+
+```
+[Interface]
+PrivateKey = [yourPrivateKey]
+Address = 10.20.10.X/32
+DNS = 10.20.50.1
+
+[Peer]
+PublicKey = [VPNServerPublicKey]
+PresharedKey = [presharedKey]
+AllowedIPs = 10.20.50.0/24, 10.20.10.X/32
+Endpoint = boulder.vpn.ethpandaops.io:51820
+PersistentKeepalive = 15
+```
+
+Note: `VPNServerPublicKey` and `presharedKey` and `X` will be provided to you in order to authenticate your device.
```

Comment:
replace the entire `Address` with `X`, otherwise X being just the last digit of the IP is easy to miss